addon/components/hover-tooltip.hbs

```diff
@@ -0,0 +1,8 @@
+<Velcro @placement="top" as |velcro|>
```

Comment:
As a getter for `placement` exists (and it returns this.args of hover-tooltip), I'm guessing the point is to be able to override the Velcro placement by adding @replacement when using hover-tooltip.
So I'm guessing this should be 
```suggestion
<Velcro @placement=this.placement as |velcro|>
```

Reply:
ooh good catch, yeah forgot to wire that up in the template :grimacing: